src/main/target/FLIGHTONE/target.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+
+#include "drivers/dma.h"
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+    DEF_TIM(TIM11, CH1,  PB9, TIM_USE_ANY,                 0, 0), //cam control
+
+    DEF_TIM(TIM8,  CH2N, PB0,  TIM_USE_MOTOR,               0, 0), // Motor 1
+    DEF_TIM(TIM8,  CH3N, PB1,  TIM_USE_MOTOR,               0, 0), // Motor 2
+    DEF_TIM(TIM5,  CH4, PA2,  TIM_USE_MOTOR,               0, 1), // Motor 3
+    DEF_TIM(TIM5,  CH3, PA3,  TIM_USE_MOTOR,               0, 0), // Motor 4
+
+    DEF_TIM(TIM3,  CH1,  PB4, TIM_USE_BEEPER,              0, 0), // Buzzer
```

Comment:
Does this board use a standard (DC) beeper, or does it use PWM to beep? If it's a DC beeper, you don't need this line (and it's probably not much use to assign a timer to this pin, since using it for something that isn't a beeper will be hard because of the on board beeper driver).

Reply:
It shorts to ground for the buzzer, so we can remove that, it was probably just copied from another target, good catch